specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-03-01-preview/managedCassandra.json

```diff
@@ -0,0 +1,1102 @@
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            },
+            "description": "Parameters specifying the managed Cassandra data center."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Accepted. The data center will be created asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
+          "200": {
+            "description": "Accepted. The data center will be updated asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "CassandraDataCenters_Update",
+        "description": "Update some of the properties of a managed Cassandra data center.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraDataCenterUpdate": {
+            "$ref": "./examples/CosmosDBManagedCassandraDataCenterPatch.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/dataCenterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DataCenterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra data center."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Update will proceed asynchronously."
+          },
+          "200": {
+            "description": "Completed synchronously. This will only happen if the fields changed by patch require no changes to the actual Cassandra cluster, such as editing the tags."
+          },
```

Comment:
No schema?  Generally this is the same schema as returned by Get and Put

Reply:
Added.